go.mod

```diff
@@ -1,7 +1,7 @@
 module github.com/pires/go-proxyproto
 
-go 1.18
```

Comment:
This breaks backwards compatibility with modules built with older Go versions, for example I can't build my service using Go 1.23 https://gitlab.com/gitlab-renovate-forks/gitlab-pages/-/jobs/9927946165